### ISSUE TITLE
Installation instructions for dedicated servers

### ISSUE THREAD

**Issue by ashwin31** [supergiant/supergiant#81]

**Comment by FestivalBobcats** [comment]
@ashwin31 sorry for the long delay on response. Could you provide a bit more information? Are you referring to on-premise servers, as opposed to AWS?

**Comment by ashwin31** [comment]
@FestivalBobcats yes, I want to use it on on-premise servers

**Comment by mclark4386** [comment]
Pretty sure he means for the backend/clustered machines, @FestivalBobcats . I was actually about to look into how hard it would be to add a DigitalOcean option instead of AWS.

**Comment by FestivalBobcats** [comment]
Yeah, I've definitely been dragging my feet on responding here... sorry about that!

@mclark4386 we actually have our [digital_ocean](https://github.com/supergiant/supergiant/tree/digital_ocean) actively underway as we speak. We're planning to have that merged in as early as tomorrow. It's only our 2nd provider, but this branch will pave the way for many more providers in much quicker iterations.

@ashwin31 you should expect to see on-premise installation instructions within the next month -- I'll get back to you if something changes.

**Comment by omniscient** [comment]
+1 for on-premise install instructions and best practices.

**Comment by phoenix24** [comment]
+1 for on-premise install instructions.

**Comment by jordanrinke** [comment]
Just an update to this - we are working with the folks over at rebar.digital to bring together a tutorial/solution that will detail how to deploy SuperGiant on hardware, and how to manage that hardware etc.

**Comment by gopherstein** [comment]
Bare metal / dedicated server installation.

**Comment by gopherstein** [comment]
Active
